### PR TITLE
SDK-159 [Java SDK] Increase test coverage up to 99%

### PR DESCRIPTION
**Pull request by robert-par**
Pull Request Checklist
======================

Please read this for our PR Culture:
------------------------------------
https://incountry.atlassian.net/wiki/spaces/ED/pages/46858869/Pull+requests+culture+Draft

Work relates to:
----------------
Link(s) to Jira Task/User Story/Bug/Epic:
- [SDK-159](https://incountry.atlassian.net/browse/SDK-159)

Code Quality (Before submitting PR)
------------
- [x] Ensure code compiles and passes Sonar Lint.
- [x] Do your best to ensure integration tests pass. If they don't then...
- [x] Remove, ignore, or mark pending any brittle tests.
- [x] Remove commented out code.

Target Release
--------------
- [ ] Please put the target release vehicle this needs to be bundled with.
- [ ] Did you increment the version? What is the new version?

### CODE REVIEW

**Comment by VladimirLevitskiy** [comment]
Please increase unit test coverage for **OAuthTokenClient.java**

**Review by VladimirLevitskiy — CHANGES_REQUESTED** [review]
Please increase unit test coverage for OAuthTokenClient.java

File: src/test/java/com/incountry/residence/sdk/http/TokenClientTest.java [hunk 1]
```diff
@@ -81,6 +81,22 @@ void defaultAuthClientPositiveTestWithoutMask() throws StorageException, IOExcep
...
+        List<String> responseList = Collections.singletonList(
+                "{'access_token'='1234567889' , 'expires_in'='1' , 'token_type'='bearer', 'scope'='" + ENV_ID + "'}"
+        );
+        TokenClient tokenClient = new OAuthTokenClient(DEFAULT_AUTH_ENDPOINT, null, ENV_ID, "<client_id>", "<client_secret>", HttpClients.createDefault());
+        int respCode = 200;
+        FakeHttpServer server = new FakeHttpServer(responseList, respCode, PORT);
+        server.start();
+        assertNotNull(tokenClient.getToken(AUDIENCE_URL, null));
+        Thread.sleep(2000);
+        assertNotNull(tokenClient.getToken(AUDIENCE_URL, null));
```
Comment (RommeroInCountry) [inline src/test/java/com/incountry/residence/sdk/http/TokenClientTest.java:96]: we need to check that tokens are not the same as well

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

**Review by VladimirLevitskiy — DISMISSED** [review]
LGTM

**Review by VladimirLevitskiy — APPROVED** [review]

**Review by RommeroInCountry — APPROVED** [review]
LGTM

**Pull request merged**